src/main/sql/example_data.sql

```diff
@@ -1,19 +1,17 @@
 --
--- Example self-administration client, please sync to the attribute values in the addon-self-administration.properties.
--- Please update all queries in this file to your needs!
--- Have to be imported in the database of the auth-server, before you deploy the addon-self-administration!
+-- DEPRECATED: replaced by extension.sql
 --
+
 INSERT INTO osiam_client (internal_id, accesstokenvalidityseconds, client_secret, expiry,
-id, implicit_approval, redirect_uri, refreshtokenvalidityseconds,
-validityinseconds)
-VALUES (10, 2342, 'super-secret', null, 'addon-self-administration-client', FALSE, 'http://localhost:8080/addon-self-administration', 4684, 1337);
+                          id, implicit_approval, redirect_uri, refreshtokenvalidityseconds,
+                          validityinseconds)
+VALUES (10, 300, 'super-secret', NULL,
```

Comment:
Actually that's a totally arbitrary number like the former `2342`. But the self admin always requests a (possibly new) access token on each call made to the `resource-server`. I've chosen `300` because it is enough time to fulfill the requests, but low enough to decrease the attack surface if someone really gets hold of the access token.